spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/deployer/strategies/HandleHealthCheckStep.java

```diff
@@ -131,9 +133,11 @@ private void deleteExistingRelease(Release existingRelease, List<String> applica
 			this.deleteStep.delete(existingRelease, existingAppDeployerData, applicationNamesToUpgrade);
 		}
 		catch (DataAccessException e) {
+			logger.debug("Error1 deleteExistingRelease {}", e);
```

Comment:
Maybe `Database Error` instead of the generic `Error1`  Why suppress this completely, e.g. not at a warning level.

Reply:
I think these exceptions only have meaning on a debug level to know if it was first or second catch. That's essentially why I added these on debug level.